common/religion/holy_sites/wc_holy_sites.txt

```diff
@@ -0,0 +1,1326 @@
+		dynasty_opinion = 15
+	}
+}
+timbermaw = {
+	county = c_timbermaw
+	
+	character_modifier = {
+		name = holy_site_timbermaw_effect_name
+		martial_per_piety_level = 1
+		garrison_size = 0.1
+	}
+}
+blackmaw = {
+	county = c_blackmaw
+	
+	character_modifier = {
+		name = holy_site_blackmaw_effect_name
+		prowess_per_stress_level = 5
+		learning = 1
+	}
+}
+
+### kaldorei_religion
+#nordrassil
+#darnassus
+astranaar = {
+	county = c_astranaar
+	
+	character_modifier = {
+		name = holy_site_astranaar_effect_name
+		stewardship = 2
+		faith_creation_piety_cost_mult = 0.3
+	}
+}
+eldrethalas = {
+	county = c_eldrethalas
+	
+	character_modifier = {
+		name = holy_site_eldrethalas_effect_name
+		martial = 1
+		stewardship = 1
+		development_growth_factor = 0.1
+	}
+}
+zin_malor = {
+	county = c_zin_malor
+	
+	character_modifier = {
+		name = holy_site_zin_malor_effect_name
+		different_culture_opinion = 10
+		learning_per_prestige_level = 1
+	}
+}
+
+### agamaggan_worship
+razorfen = {
+	county = c_razorfen
+	
+	character_modifier = {
+		name = holy_site_razorfen_effect_name
+		plains_advantage = 5
+		happy_powerful_vassal_levy_contribution_mult = 0.25
+	}
+}
+downs = {
+	county = c_downs
+	
+	character_modifier = {
+		name = holy_site_downs_effect_name
+		monthly_dynasty_prestige_mult = 0.05
+	}
+}
+razor_hill = {
+	county = c_razor_hill
+	
+	character_modifier = {
+		name = holy_site_razor_hill_effect_name
+		martial = 1
+		stewardship_per_prestige_level = 1
+	}
+}
+#zin_malor
+#nordrassil
+
+################
+# light_group
+################
+### holy_light
+tyrs_hand = {
+	county = c_tyrs_hand
+	
+	character_modifier = {
+		name = holy_site_tyrs_hand_effect_name
+		tolerance_advantage_mod = 2
+		defender_advantage = 2
+		same_faith_opinion = 2
+	}
+}
+northshire = {
+	county = c_northshire
+	
+	character_modifier = {
+		name = holy_site_northshire_effect_name
+		negate_health_penalty_add = 0.5
+		negate_fertility_penalty_add = 0.05
+	}
+}
+# For unknown reason, stormwind can't be holy site
+# stormwind = {
+	# county = c_stormwind
+	
+	# character_modifier = {
+		# name = holy_site_stormwind_effect_name
+		# development_growth_factor = 0.1
+		# stewardship = 1
+	# }
+# }
+stromgarde = {
+	county = c_stromgarde
+	
+	character_modifier = {
+		name = holy_site_stromgarde_effect_name
+		martial_per_piety_level = 1
+		prowess = 2
+	}
+}
+gilneas = {
+	county = c_gilneas
+	
+	character_modifier = {
+		name = holy_site_gilneas_effect_name
+		castle_holding_build_gold_cost = -0.2
+		happy_powerful_vassal_tax_contribution_mult = 0.2
+	}
+}
+ironforge = {
+	county = c_ironforge
+	
+	character_modifier = {
+		name = holy_site_ironforge_effect_name
+		mountains_cancel_negative_supply = yes
+		stewardship = 1
+	}
+}
+
+### scarlet_light
+#tyrs_hand
+lordaeron = {
+	county = c_lordaeron
+	
+	character_modifier = {
+		name = holy_site_lordaeron_effect_name
+		diplomacy_per_piety_level = 1
+		monthly_piety_gain_per_knight_add = 0.05
```

Comment:
Don't work

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/99658649-d862f600-2a70-11eb-9844-698b812d1d69.png)

</details>

Reply:
Changed in 0b8e129ed81ceb91b14b5b7a7c20e749925434a3.